Makefile.stsv1alpha

```diff
@@ -0,0 +1,66 @@
+.PHONY: all pull fetch patch generate clean codegen mod docs move-other patch-post fmt test stage
+
+include Makefile
+
+PACKAGE_NAME=stsv1alpha
+SPEC_BASE_URL:=https://docs.equinix.com/api-catalog/stsv1alpha
+SPEC_ROOT_FILE:=openapi.yaml
+
+CODE_DIR=${CODE_BASE_DIR}/${PACKAGE_NAME}
+TEMPLATE_DIR=${TEMPLATE_BASE_DIR}/${PACKAGE_NAME}
+SPEC_FETCHED_DIR=${SPEC_BASE_DIR}/${PACKAGE_NAME}/oas3.fetched
+SPEC_PATCH_DIR=${SPEC_BASE_DIR}/${PACKAGE_NAME}/patches
+SPEC_PATCHED_DIR=${SPEC_BASE_DIR}/${PACKAGE_NAME}/oas3.patched
+
+all: pull fetch patch generate stage
+
+generate: clean codegen remove-unused patch-post mod fmt test
+
+pull:
+	${CRI} pull ${OPENAPI_IMAGE}
+
+fetch:
+	mkdir -p ${SPEC_FETCHED_DIR}
+	curl -o ${SPEC_FETCHED_DIR}/${SPEC_ROOT_FILE} ${SPEC_BASE_URL}/${SPEC_ROOT_FILE}
```

Comment:
Note that, as of now, the spec for this package is not published in the API catalog, so this task will fail.  Not a huge problem because it requires manually running the spec sync workflow anyway, but might be good to add a note here and/or comment out the `curl` command so others don't expect this to work.